lustrefs-exporter/benches/jobstats.rs

```diff
@@ -37,24 +40,42 @@ job_stats:{}"#,
     str_repeat!(JOBSTAT_JOB, 1000)
 );
 
-async fn parse_synthetic_yaml(input: &'static str) {
+async fn parse_synthetic_yaml_otel(input: &'static str) {
+    // Set up OpenTelemetry metrics
+    let registry = Registry::new();
+    let exporter = opentelemetry_prometheus::exporter()
+        .with_registry(registry.clone())
+        .build()
+        .unwrap();
```

Comment:
Unwrap

Reply:
This is benches/tests, unwrap shouldn't be an issue here